### PR TITLE
Android Design Library Version Updated

### PR DESCRIPTION
**Pull request by pratikbutani**
I have updated android design library with new version. We know that before updating this library we are getting issue like: http://stackoverflow.com/q/30540648/1318946

After updated library version i have posted answer here: http://stackoverflow.com/a/31488708/1318946

### CODE REVIEW

**Comment by jaredsburrows** [comment]
`+2,036  −1`?

This is a "dirty" pull request. Please remove your user IDE generated files in the `./idea` folder.

**Comment by johnjohndoe** [comment]
@jaredsburrows +1

File: app/build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ repositories {
 }
 
 dependencies {
-    compile 'com.android.support:design:22.2.0'
+    compile 'com.android.support:design:22.2.1'
     compile 'com.android.support:appcompat-v7:22.2.0'
```
Comment (GautamGupta) [inline app/build.gradle]: All 4 of the libraries here can be 22.2.1.

**Comment by pratikbutani** [comment]
@jaredsburrows @johnjohndoe Sorry for that. No what can i do. I cant Understand.

**Comment by jaredsburrows** [comment]
@pratikbutani Please remove your user IDE generated files in the  `./idea`  folder.

**Comment by PareshMayani** [comment]
This PR was dirty and now not of any use as Chris has already made this change https://github.com/chrisbanes/cheesesquare/commit/3a4aca1040c0d37dfd6f0edd189dba80f58df16a

**Comment by jaredsburrows** [comment]
Yes, now this PR has conflicts.

**Comment by jaredsburrows** [comment]
Also, @chrisbanes already updated this: https://github.com/chrisbanes/cheesesquare/blob/master/app/build.gradle#L28

**Comment by OCman** [comment]
And I think I found another bug. https://code.google.com/p/android/issues/detail?id=182345&colspec=ID%20Type%20Status%20Owner%20Summary%20Stars

**Comment by brucewuu520** [comment]
@chrisbanes 
1. use TabLayout  in fragment has bugs, label do not display 
2. app:tabTextAppearance in xml is useless

**Comment by GautamGupta** [comment]
@brucewuu520 Can you post sample code?

**Comment by brucewuu520** [comment]
fragment view xml:
<?xml version="1.0" encoding="utf-8"?>
<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@+id/main_content"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <android.support.design.widget.AppBarLayout
        android:id="@+id/appbar"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar">

        <include
            layout="@layout/main_appbar"
            android:layout_width="match_parent"
            android:layout_height="?attr/actionBarSize"
            app:layout_scrollFlags="scroll|enterAlways" />

        <android.support.design.widget.TabLayout
            android:id="@+id/tabs"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            app:tabIndicatorColor="?android:attr/textColorPrimary"
            app:tabTextAppearance="@style/Tab.TextAppearance" />

    </android.support.design.widget.AppBarLayout>

    <android.support.v4.view.ViewPager
        android:id="@+id/viewpager"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        app:layout_behavior="@string/appbar_scrolling_view_behavior" />
</android.support.design.widget.CoordinatorLayout>

the fragment code:

 MenuAdapter adapter = new MenuAdapter(getChildFragmentManager());
        adapter.addFragment(new DoctorsHallFragment(), getString(R.string.doctors_hall));
        adapter.addFragment(new FamilyDoctorFragment(), getString(R.string.family_doctor));
        adapter.addFragment(new ExaminationFragment(), getString(R.string.examination));
        adapter.addFragment(new HealthCareFragment(), getString(R.string.health_care));

        mViewPager.setAdapter(adapter);
        tabs.setupWithViewPager(mViewPager);

**Pull request closed without merging**